### PR TITLE
cap3: new package

### PR DESCRIPTION
**Pull request by akthoma**
adding new package cap3

### CODE REVIEW

File: var/spack/repos/builtin/packages/cap3/package.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+#     spack install cap3
+#
+# You can edit this file again by typing:
+#
+#     spack edit cap3
+#
+# See the Spack documentation for more information on packaging.
+# If you submit this package back to Spack as a pull request,
+# please first remove this boilerplate and all FIXME comments.
+#
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/cap3/package.py]: Don't forget to remove this template (but keep the copyright).

File: var/spack/repos/builtin/packages/cap3/package.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+from spack import *
+
+
+class Cap3(Package):
+    """CAP3 is DNA Sequence Assembly Program"""
+
+    homepage = "http://seq.cs.iastate.edu/"
+    url      = "http://seq.cs.iastate.edu/CAP3/cap3.linux.x86_64.tar"
+
+    version('3', '5393d937978ecc5f18fcb741140e1f02')
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/cap3/package.py]: It looks like CAP3 is the name of the package, so the version isn't really 3. I guess they don't provide versioned URLs? That's annoying. As soon as they issue a new release, this checksum is going to change.

Comment (akthoma) [inline var/spack/repos/builtin/packages/cap3/package.py]: What do you suggest I do? It looks like last time there was an update was 2015-02-11. I can use that as the version number, or go a different route if you have something else in mind

Comment (adamjstewart) [inline var/spack/repos/builtin/packages/cap3/package.py]: I like the date idea, let's do that.

**Pull request merged**